.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,19 +1,42 @@
 ---
-name: Bug report
-about: Bug report
+name: Bug Report
+about: Report an unexpected behaviour that you have encountered.
 title: ''
 labels: bug
 assignees: ''
 
 ---
 
-**V version:** 
-**OS:** 
+**V Version:**
 
-**What did you do?**
+**OS:**
 
+**Source Code to reproduce the bug:**
```

Comment:
not all bugs have source code

in general, I like keeping things simple for people, the fewer instructions/questions, the better